spec/acceptance/card_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource "Cards" do
+  fixtures :all
+  Scenic.database.refresh_materialized_view(:unified_restrictions, concurrently: false, cascade: false)
+  Scenic.database.refresh_materialized_view(:unified_cards, concurrently: false, cascade: false)
+  Scenic.database.refresh_materialized_view(:unified_printings, concurrently: false, cascade: false)
+
+  header "Content-Type", "application/json"
+  header "Host", "api-preview.netrunnerdb.com"
+
+  get "/api/v3/public/cards" do
+    example_request "All Cards" do
+      expect(status).to eq 200
+    end
+  end
+
+  get "/api/v3/public/cards/:id" do
+    route_summary 'Retrieve a single Card by ID'
+    parameter :id, type: :string, required: true
+
+    let(:id) { 'hedge_fund' }
+    example_request "Get A Single Card" do
+      expect(status).to eq 200
+    end
+  end
```

Comment:
I don't really know how this testing framework works but this looks like a no-op test. I think that checking that an endpoint returns 200 is meaningless if it's returning the wrong thing.